README.rst

```diff
@@ -298,13 +298,13 @@ add significant overhead. When versioning is not required, ``ValueSerializer`` i
 
   public static final Serializer<Boolean> BOOLEAN = new ValueSerializer<Boolean>() {
       @Override
-      protected void serializeValue(@NotNull SerializerOutput output, @NotNull Boolean object) throws IOException {
+      protected void serializeValue(SerializationContext context, @NotNull SerializerOutput output, @NotNull Boolean object) throws IOException {
```

Comment:
Ah thanks! Do you mind adding the `@NotNull` annotations as well, for consistency?

Reply:
oh,I'll check it.